d3a_api_client/redis_aggregator.py

```diff
@@ -55,7 +55,6 @@ def _batch_response(self, message):
         data = json.loads(message['data'])
         if self.aggregator_uuid != data['aggregator_uuid']:
             return
-        logging.info(f"Market Stats. Information: {message}")
```

Comment:
Any reason to remove it?

Reply:
I removed it because it seems a copy paste remnant (look at the message, it says "Market Stats")
This is just the response of the batch command, so IMO it is ok to remove this line. OK?

Reply:
I would rather leave it as a debug log with improved message structure rather than completely skipping it.